bin/parallel_cucumber

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
Do we need this? I don't think we have an cucumber

Reply:
Oversight, thanks! Updated in https://github.com/bikeindex/bike_index/pull/866/commits/af3c4acce13167d632d03c9c842a4091522391ab https://github.com/bikeindex/bike_index/pull/866/commits/ec2fe3c03a39dc83eacf1841f3a16517486fad0b and https://github.com/bikeindex/bike_index/pull/866/commits/896be8f95d375d018ab51356db4ec7de040d95f4